templates/developer.rackspace.com/_includes/rackspace-client.html

```diff
@@ -5,5 +5,5 @@
     t.async = true;
     h = d.getElementsByTagName('script')[0];
     h.parentNode.insertBefore(t,h);
-  })(document, '//www.rackspace.com/data/client');
+  })(document, '//www.rackspace.com/api/geo.jsonp?callback=Array.prototype.push');
```

Comment:
@bighappyface If this is just a plain JSONP payload, couldn't we simplify this by explicitly creating the tag with the `async` attribute? 

```html
<script src="//www.rackspace.com/api/geo.jsonp?callback=Array.prototype.push" async></script>
```